snapshot/btrfs/package.go

```diff
@@ -0,0 +1,7 @@
+package btrfs
```

Comment:
i added this because the root-test Makefile target expands TEST_REQUIRES_ROOT_PACKAGES using 'git grep -l testutil.RequiresRoot' and ends up including snapshot/btrfs on the "go test" command line.  given that git is not go build tag aware i sure how i could filter this entry out.  but playing around with the Makefile a bit more i figured out that if i cleanup TEST_REQUIRES_ROOT_PACKAGES i can $(filter ...) it against $(PACKAGES) to remove packages not supported on the current platform.  this allows me to delete snapshot/btrfs/package.go and snapshot/overlay/package.go.